common/configurable/src/main/java/io/helidon/common/configurable/ResourceUtil.java

```diff
@@ -99,4 +105,47 @@ static InputStream toIs(URI uri, Proxy proxy) {
             throw new ResourceException("Failed to open strem to uri: " + uri, e);
         }
     }
+
+    static Optional<Resource> fromConfigPath(Config config, String keyPrefix) {
```

Comment:
These methods basically define some conventions for config key suffixes and types.
I can see this being used internally to have consistency between all the different configurable entities, but I don't understand why we would expose the keyPrefix in our API.

Reply:
We should have a look at this separately. This PR is to align with development guidelines - this library is in GIT for a long time. I agree we may need to refactor this, though not as a part of this PR.

Reply:
Roger that.